README.md

```diff
@@ -1,2 +1,18 @@
 # BonnyCI Documentation
 [![Build Status](https://travis-ci.org/BonnyCI/docs.svg?branch=master)](https://travis-ci.org/BonnyCI/docs)
+
+## Table of Contents
+- [Introduction](#introduction)
+- [Getting Started](#getting-started)
+- [Contributing](#contributing)
+
+## Introduction
+BonnyCI is a Continuous Integration service offered to developers of Open Source projects.
+
+## Getting Started
+Checkout the [getting started](getting_started) section.
+
+## Contributing
```

Comment:
So, I think we should go ahead and start dogfooding some of the existing documentation like [this](https://github.com/BonnyCI/hoist/blob/master/CONTRIBUTING.md). It might also behoove use to mention [basic github flow](https://guides.github.com/introduction/flow/) and [how to write a commit message](http://chris.beams.io/posts/git-commit/), but that depends on how elementary we want to get with this.